packages/search-in-workspace/src/browser/search-in-workspace-widget.tsx

```diff
@@ -248,8 +248,10 @@ export class SearchInWorkspaceWidget extends BaseWidget implements StatefulWidge
 
     protected onUpdateRequest(msg: Message): void {
         super.onUpdateRequest(msg);
-        ReactDOM.render(<React.Fragment>{this.renderSearchHeader()}{this.renderSearchInfo()}</React.Fragment>, this.searchFormContainer);
-        this.onDidUpdateEmitter.fire(undefined);
+        if (this.renderSearchInfo()) {
+            ReactDOM.render(<React.Fragment>{this.renderSearchHeader()}{this.renderSearchInfo()}</React.Fragment>, this.searchFormContainer);
+            this.onDidUpdateEmitter.fire(undefined);
```

Comment:
it was called before always, not only sometimes. Does it change semantic for some clients?

Reply:
It would render as long as the *searchInfo* is correct (i.e. it only excludes the case **xx results in 0 file**, which caused the glitch previously). The change should not have impacts on how the component and the search function is used.